### PR TITLE
Change rendering of alpine Hut to vector icon

### PR DESCRIPTION
**Pull request by giggls**
Hello,

while trying to change the @transportation-icon color for german style I stumbled upon the fact that "alpine_hut" was still a raster icon and some inconsistencies in amenity-points.mss.

Thus I made up a vector version of alpine_hut and fixed the inconsistencies.

This is what this pull-request is all about.

### CODE REVIEW

**Comment by kocio-pl** [comment]
Could also you drop here some before/after rendering example?

File: amenity-points.mss [hunk 1]
```diff
@@ -24,17 +24,16 @@
 
 /* Note that .points is also used in water-features.mss */
 .points {
-  [feature = 'tourism_alpine_hut'][zoom >= 13] {
-    point-file: url('symbols/alpinehut.p.16.png');
```
Comment (sommerluk) [inline amenity-points.mss]: As alpinehut.p.16.png would not be used anymore after this PR, the file should be deleted.

**Comment by giggls** [comment]
I doubt that the differences are all that visible, but here we go:
5631-new
5631-old

File: amenity-points.mss (hunk 1 above)
Comment (giggls) [inline amenity-points.mss]: OK, branch with pull request updated.

**Comment by kocio-pl** [comment]
You would be surprised how many times pixel peeping shows general problems with rendering... But it looks like an improvement for me, thanks for this contribution!

Small details to be fixed:
- please join the paths (Path > Union) and save it as a plain SVG (instead of "Inkscape SVG")
- you should merge all these commits into one

**Comment by giggls** [comment]
Looks like my knowledge of git stops here. I just tried to merge the 3 commits with the result that I have 5 now instead of 3.

**Comment by kocio-pl** [comment]
I'm also no git expert, so I use gitk for mixed reset, to better control what I'm doing, however this can be done also with CLI (https://git-scm.com/docs/git-reset#git-reset---mixed). After this I add and commit files as usual.

**Comment by giggls** [comment]
*argh* looks like deleting and recreating of the branch with just one commit was not a very good idea :(

**Comment by imagico** [comment]
By the way IIRC if you have locally deleted and re-created a branch you can force-push it to your PR-branch on github without problems (`git push -f`)

**Comment by kocio-pl** [comment]
> while trying to change the @transportation-icon color for german style

Is German style switching to this new fork?

**Comment by giggls** [comment]
Switching to which fork? German style _is_ a fork of openstreetmap-carto. If I have changes which are unrelated to German style like this one I will always try to get them upstream. Because this will make it easier for me to keep German style master in sync to upstream.

**Comment by kocio-pl** [comment]
I thought it still uses XML-based version.

**Pull request closed without merging**